gradle-baseline-java-config/resources/findbugs/excludeFilter.xml

```diff
@@ -18,4 +18,16 @@
             <Bug pattern="URF_UNREAD_PUBLIC_OR_PROTECTED_FIELD"/> <!-- @Rule ivars can be public and unused -->
         </Or>
     </Match>
+
+    <!-- Ignore the following bug patterns in Immutables generated code -->
+    <!-- (i.e., classes beginning with 'Immutable' and inner classes of same) -->
+    <Match>
+        <Class name="~.*Immutable.*(\$.*)?" />
```

Comment:
unfortunately upgrading didn't do what I wanted since it only adds @SuppressFBWarnings when the findbugs annotations jar is on the classpath. Since not everyone uses Immutables, I'll just add this block to the excludeFilter.xml in my project.